Virtual Keyboard

FrostyDog opened the pull request:
1. Task:
https://github.com/rolling-scopes-school/tasks/blob/master/tasks/virtual-keyboard/virtual-keyboard-en.md
2. Screenshot
![image](https://user-images.githubusercontent.com/20482331/83155538-a1cfa280-a101-11ea-980f-f9eb7b5af930.png)
3. Deploy
https://frostydog.github.io/virtual-keyboard/

4. Done 28.05.20 (deadline 28.05.20)
5. Score: 200 / 200
- [x] Minimal scope
    - [x] Feature 1 is implemented (20/20)
    - [x] Feature 2 is implemented (5/10)
- [x] Basic scope
    - [ ] Feature 3 is implemented (0/15)
    - [x] Feature 4 is implemented (7/15)
- [x] Extra scope
    - [x] Feature 5 is implemented (12/15)
- [x] Technical requirements
    - [x] ESLint is used (15/15)
    - [x] ES6 features are used (15/15)
    - [x] Requirements to the repository, commits and pull request are met (10/10)
- [ ] Penalties --??
    - [ ] Errors while app's working. (-10 for each, but no more than the total number of points for requirement implementation)
    - [ ] Not comply with the requirements for Pull Request, repository, commits names (-10) scores by mentor

kat-ya commented on package.json:
```diff
@@ -0,0 +1,29 @@
+    "eslint-plugin-import": "^2.20.2"
+  },
+  "dependencies": {
+    "lint": "^0.7.0"
```
are you sure `lint` should be a dependency? 
I see youhave eslint as a devDependency, so `lint` seems redundant and might increase bundle size

FrostyDog replied:
Perfectly true, fixed. Thanks!

kat-ya commented on package.json:
```diff
@@ -0,0 +1,29 @@
+  "main": "index.js",
+  "scripts": {
+    "test": "npm run lint",
+    "lint": "node node_modules/eslint/bin/eslint.js ./modules"
```
usually once you install a package like
`npm i --save-dev eslint`
you can use it in `scripts` like
`eslint src`

FrostyDog replied:
Understood, did not had a `eslint-cli installed` , so `eslint` coomand was not recognizable.
Fixed!3

FrostyDog commented:
> @FrostyDog ,
> there seems to be some problem with the demo of the functionality of a keyboard on https://frostydog.github.io/virtual-keyboard/ on my computer. looks like textarea.js is missing
> <img alt="Screen Shot 2020-05-31 at 3 48 28 PM" width="1440" src="https://user-images.githubusercontent.com/56261277/83350579-7396d080-a356-11ea-8891-3df1617b623f.png">
> 
> can you, please, take a look at it

Took a look and fixed. Paths were wrong, I suppose.
Updated gh-pages and dev.
Thank you for checking ! :)
